src/main/groovy/com/mini/asaas/payment/PaymentStatus.groovy

```diff
@@ -4,10 +4,16 @@ import com.mini.asaas.utils.MessageSourceUtils
 
 enum PaymentStatus {
 
-    CANCELED,
-    OVERDUE,
-    PENDING,
-    RECEIVED
+    CANCELED("secondary"),
```

Comment:
Não sei se adicionar o tema, algo específico do front-end, neste enum aqui seja tão interessante.
E mesmo se for, seria melhor criar um enum para tal, ao invés de usar as strings por si só. Talvez um enum `AtlasTheme`.

Outra maneira seria utilizar a tag lib para processar o valor do status do payment e, a partir do valor, inserir o tema correto.

Reply:
Obrigada